diagnostics/tropical_pacific_sea_level/spherical_area.py

```diff
@@ -0,0 +1,74 @@
+import numpy as np
+
+def cal_area(lon,lat,dlon,dlat,r_earth = 6.371*1E8) :
+# input : lon, lat (not array, but single element)
+# output : area that matches the [lon,lat] coordinate 
+#          units in cm^2
+
+    # constant
+    arad = r_earth
+    dlon = np.float64(dlon*np.pi/180.)
+    dlat = np.float64(dlat*np.pi/180.)
+    lat = (90.-lat)*np.pi/180.
+    area = arad*arad*np.sin(lat)*dlon*dlat
+
+    return area
+
+
+
+def da_area(da_var, lonname='lon', latname='lat', xname='x', yname='y', model=None, r_earth = 6.371*1E8):
+    """
+    calculate spherical area
+    
+    """
+#     r_earth = 6.371*1E8         # cm
+
+    if model in ['gfdl']:
```

Comment:
Testing on a hard-coded model name isn't extensible, and it obscures the real condition being tested for here. Is the difference in dimensionality in dx, dy between the two branches an areacello vs. volcello distinction?